packages/angular/cli/lib/cli/index.ts

```diff
@@ -5,25 +5,38 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import { terminal } from '@angular-devkit/core';
 import { createConsoleLogger } from '@angular-devkit/core/node';
+import * as colors from 'ansi-colors';
```

Comment:
What is the reason for changing to use `ansi-color` instead of `@angular-devkit/core`?

Reply:
The style removal capability is needed to support future work.  The cross-platform symbol capabilities will also be useful for future improvements to user messaging ([code link](https://github.com/doowb/ansi-colors/blob/master/symbols.js)).  Since it's already being imported there isn't a compelling reason to import another module to also provide color support if one happens to already be present.
It also doesn't have the bugs that are present currently.  Try the following for instance:
```
const { terminal } = require('@angular-devkit/core');
console.log(terminal.bold(`foo ${terminal.red(terminal.dim('bar'))} baz`));
```
Versus:
```
const color = require('ansi-colors');
console.log(color.bold(`foo ${color.red(color.dim('bar'))} baz`));
```
![image](https://user-images.githubusercontent.com/19598772/59555848-6044f680-8f87-11e9-84c8-e87189162519.png)